src/apps/search-result/search-result.tsx

```diff
@@ -39,7 +39,12 @@ const SearchResult: React.FC<SearchResultProps> = ({ q, pageSize }) => {
   const [resultItems, setResultItems] = useState<Work[]>([]);
   const [hitcount, setHitCount] = useState<number>(0);
   const [canWeTrackHitcount, setCanWeTrackHitcount] = useState<boolean>(false);
-  const { PagerComponent, page } = usePager(hitcount, pageSize);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
Please consider if we really need a separate loading state.

As I see it this is already provided by React Query and `useSearchWithPaginationQuery`. I do not understand why we need a separate value.